index.html

```diff
@@ -726,85 +822,43 @@ <h4>Automation via Workflows</h4>
 }
         </pre>
 
-      </section>
-
-    </section>
-
-    <section>
-      <h3>Verifying</h3>
-      <p>
-The following APIs are defined for verifyig a Verifiable Credential:
-      </p>
-
-      <table class="simple api-summary-table"
-        data-api-path="/credentials/verify /presentations/verify"></table>
-
-      <section>
-        <h4>Verify Credential</h4>
-        <p>
-        </p>
-
-        <div class="api-detail"
-          data-api-endpoint="post /credentials/verify"></div>
-      </section>
-
-      <section>
-        <h4>Verify Presentation</h4>
-        <p>
-        </p>
-
-        <div class="api-detail"
-          data-api-endpoint="post /presentations/verify"></div>
-      </section>
-
-    </section>
-
-    <section>
-      <h3>Presenting</h3>
-      <p>
-The following APIs are defined for presenting a Verifiable Credential:
-      </p>
-
-      <table class="simple api-summary-table"
-        data-api-path="
-          /credentials/derive /presentations/prove
-          /presentations/available /presentations/submissions"
-        ></table>
-
       <section>
-        <h4>Derive Credential</h4>
-        <p>
+        <h4>Start Workflow</h4>
+        <p class="issue" title="Generalize 'Presentation Availability' endpoint">
+The existing <a href="#presentation-availability">Presentation Availability</a>
+endpoint requires a Verifiable Presentation to be submitted to the endpoint.
+The <em>Start Workflow</em> endpoint defined here generalizes  the <a
+href="#presentation-availability">Presentation Availability</a> endpoint by
+allowing arbitrary data to be posted to the endpoint. It is suggested that we
+can merge the API endpoint defined in this section with the <a
+href="#presentation-availability">Presentation Availability</a> endpoint via
+the generalization provided in this section. The term "workflow" is a working
+title and is not intended to be the final feature name.
         </p>
 
         <div class="api-detail"
-          data-api-endpoint="post /credentials/derive"></div>
+          data-api-endpoint="post /workflows/{name}/start"></div>
```

Comment:
embedding name like this will limit its potential to URL safe characters.... prefer to see it as a post body argument.

Reply:
Yes, understood. Being tracked in this issue: https://github.com/w3c-ccg/vc-api/issues/256

This will be a topic of discussion on the next VC API call.